articles/terraform/create-linux-virtual-machine-with-infrastructure.md

```diff
@@ -261,9 +261,15 @@ To use SSH to connect to the virtual machine, do the following steps:
 1. Use SSH to connect to the virtual machine.
 
     ```console
-    ssh -i id_rsa azureuser@<public_ip_address>
+    chmod 600 ~/id_rsa
+    ssh -i ~/id_rsa azureuser@<public_ip_address>
     ```
 
+## Clean up resources
```

Comment:
After I walked this article to the end, I really miss this clean-up session to destroy everything tf have created before.